docs/source/configuration/volto-slate/configuration-settings.md

```diff
@@ -0,0 +1,167 @@
+---
+html_meta:
+  "description": "Slate Editor configuration settings"
+  "property=og:description": "Slate Editor configuration settings"
+  "property=og:title": "Editor Configuration"
+  "keywords": "Volto, Plone, frontend, React, Slate, Slate-React, volto-slate"
+---
+
+# Editor Configuration
+
+You can customize how `volto-slate` editor behaves and its features.
+These are controlled from the `config.js` settings.
+
+## `defaultBlockType`
+
+The default block type for a website.
+
+```js
+config.settings.defaultBlockType = 'slate';
+```
+
+## `slate.toolbarButtons`
+
+This setting contains the default enabled Slate `inlineToolbar` buttons.
+
+```js
+config.settings.slate.toolbarButtons = [
+    'bold',
+    'italic',
+    'strikethrough',
+    'link',
+...
+  ];
+```
+
+## `slate.buttons`
+
+The Slate toolbar button components.
+
+```js
+settings.slate.buttons = [
+    ...settings.slate.buttons,
+    'block-quote': <BlockButton ... />
```

Comment:
@nileshgulia1 my editor does not like this syntax, indicating that `...` is not valid. Please provide either valid syntax or the correct lexer for this code block.

Reply:
I have replaced code blocks which involve React Elements to use `jsx` lexer.